### DISCUSSION TITLE
`F` at a dir to fuzzy find in this directory

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvim-neo-tree/neo-tree.nvim#1160] [Ideas]
Like `Find in files` when right click the dir node in IDEA

<img width="477" alt="image" src="https://github.com/nvim-neo-tree/neo-tree.nvim/assets/95092244/bbfc4288-ce59-4e0c-a17c-a09b2487ca68">

**Comment by cseickel** [comment]
"Find in Files..." is a grep within the files, not a search for files by name like the fuzzy finding in neo-tree.  I think Telescope and others already do a fine job of this and we won't be re-implementing that here.

If what you actually want is a fuzzy search on file names within a sub directory, just set this option:
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/71f1841ba6c652908678cece623f52c1fea8a6cd/lua/neo-tree/defaults.lua#L483

OR use the "fuzzy_sorter" command which is mapped to `#` by default (but I remap it to `f` because I use it all the time).

With either of those options, you could just include the path name in your search term to filter within that sub folder. In your example above:

```
s c
```

would probably be enough to fuzzy find `lua/scratch/config`
